byterun/caml/misc.h

```diff
@@ -149,10 +149,10 @@ extern void caml_set_fields (intnat v, unsigned long, unsigned long);
 
 /* snprintf emulation for Win32 */
 
-#ifdef _WIN32
+#if defined(_WIN32) && defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
For Mingw ports, I guess that _WIN32 is defined but not _MSC_VER -- is that right?  Then this patch will disable the snprintf emulation for these ports.  Is that ok?